Makefile

```diff
@@ -2,7 +2,8 @@
 LDFLAGS = $(if $(DEBUGGER),,-s -w) $(shell ./hack/version.sh)
 
 # SET DOCKER_REGISTRY to change the docker registry
-DOCKER_REGISTRY := $(if $(DOCKER_REGISTRY),$(DOCKER_REGISTRY),localhost:5000)
+DOCKER_REGISTRY_PREFIX := $(if $(DOCKER_REGISTRY),$(DOCKER_REGISTRY)/,)
```

Comment:
Maybe in develop environment, we want to directly build image on pingcap/chaos-mesh, so that we will not need to modify config in helm to test.

Reply:
Aha, it's the responsibility for privileged people to know what he is doing with `make docker-push`. 

Even without this Makefile, you can push to product env direcly by typing every `docker push` command. If you don't want this, set a `DOCKER_REGISTRY` variable.